src/api/payment-requests/payment-requests.md

```diff
@@ -467,12 +467,168 @@ them to find the Payment Request and proceed to pay.
 {% endjson %}
 
 
+### Pay a Payment Request by Id
+
+There are two methods of paying a payment request.
+The first uses Centrapay [Assets] and requires you to provide the Id of the asset to use for payment.
+Alternatively you can provide a transaction hash and the Centrapay [Asset Type] for any crypto payments that we support.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/payment'
+  auth 'jwt'
+  example {
+    title 'Pay a Payment Request with a Centrapay asset'
+    body ({
+      "assetId": "WRhAxxWpTKb5U7pXyxQjjY"
+    })
+  }
+
+  example {
+    title 'Pay a Payment Request using transaction hash from a crypto payment'
+    body ({
+      "assetType": "bitcoin.main",
+      "transactionHash": "VMXMkUttDGTVz4ESv5ND56",
+    })
+  }
+
+{% endreqspec %}
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "MhocUmpxxmgdHjr7DgKoKw",
+  "url": "https://app.centrapay.com/pay/MhocUmpxxmgdHjr7DgKoKw",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantName": "Centrapay Café",
+  "configId": "5efbe2fb96c08357bb2b9242",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000"
+  },
+  "paymentOptions": [
+    {
+      "amount": "1000",
+      "assetType": "centrapay.nzd.main"
+    },
+    {
+      "amount": "1000",
+      "assetType": "cca.coke.main"
+    }
+  ],
+  "status": "paid",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "updatedAt": "2021-06-08T04:04:27.426Z",
+  "expiresAt": "2021-06-08T04:06:27.426Z",
+  "liveness": "main",
+  "expirySeconds": 120,
+  "paidBy": {
+    "assetTotals": [
+      {
+        "type": "centrapay.nzd.main",
+        "description": "Centrapay NZD",
+        "settlementDate": "2021-06-28T04:04:27.426Z",
+        "total": {
+          "amount": "550",
+          "currency": "NZD"
+        }
+      }
+    ]
+  }
+}
+{% endjson %}
+
+### Refund a Payment Request by Id
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/refund'
+  auth 'jwt'
+  example {
+    title 'Refund a Payment Request'
+    body ({
+      "value": {
+        "amount": "100",
+        "currency": "NZD",
+      },
+      "externalReference": "2021-12-01T01:31:46+00:00",
+    })
+  }
+
+{% endreqspec %}
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "MhocUmpxxmgdHjr7DgKoKw",
+  "url": "https://app.centrapay.com/pay/MhocUmpxxmgdHjr7DgKoKw",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantName": "Centrapay Café",
+  "configId": "5efbe2fb96c08357bb2b9242",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000"
+  },
+  "paymentOptions": [
+    {
+      "amount": "1000",
+      "assetType": "centrapay.nzd.main"
+    },
+    {
+      "amount": "1000",
+      "assetType": "cca.coke.main"
+    }
+  ],
+  "status": "paid",
```

Comment:
Copy-pasta error here? Should this be `refunded` (or whatever we set it to)?

Reply:
This is something we will need to discuss. This idea of paid being a final status has been around with the legacy payment flow. If we add other statuses a fully refunded payment request can just be refunded but partially refunded payments I'm not sure what it would be.

Reply:
Nathan and I decided to keep the status as paid, so that we aren't adding any complexity to our API contract.

Reply:
Yeah sure thing - I'm happy with that now. Will revisit once we get further into partial payments I'm sure, it seems like something that would be useful to enable consumers to track themselves